platformio/compat.py

```diff
@@ -58,7 +58,7 @@ def is_bytes(x):
     def path_to_unicode(path):
```

Comment:
This returns unicode if it already is unicode, but string if it isn't? Seemed like a bug to me, and it doesn't seem to be used anywhere yet, so I changed it to always return unicode.